### PR TITLE
Add Microsoft Application Insights

### PR DESCRIPTION
**Pull request by hkjels**
New package:

**Extern:** Generated.

Update:

**Extern:** The API did not change.

### CODE REVIEW

File: applicationinsights/build.boot [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      :scm         {:url "https://github.com/cljsjs/packages"}
+      :description "Microsoft Application Insights JavaScript SDK"
+      :url         "https://azure.microsoft.com/en-us/services/application-insights/"
+      :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+   (download :url (str "https://github.com/Microsoft/ApplicationInsights-JS/archive/master.zip")
+             :unzip true)
+   (sift :move {(re-pattern (str "ApplicationInsights-JS-master/dist/ai.js"))   "cljsjs/development/applicationinsights.inc.js"
```
Comment (Deraen) [inline applicationinsights/build.boot]: Best would be to add the library name to the path before development/production part:

`"cljsjs/applicationinsights/development/applicationinsights.inc.js"`

https://github.com/cljsjs/packages/wiki/Creating-Packages#6-writing-the-package-task

File: applicationinsights/build.boot [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                (re-pattern (str "ApplicationInsights-JS-master/dist/ai.0.js")) "cljsjs/production/applicationinsights.min.inc.js"})
```
Comment (Deraen) [inline applicationinsights/build.boot]: Is `0` the minified file of the latest version or what does it mean? The dist folder in zip seems to have all versions and in most cases there is `ai.version.js` and `ai.version.min.js` files.

File: applicationinsights/boot-cljsjs-checksums.edn [hunk 3]
```diff
@@ -0,0 +1,2 @@
+{"cljsjs/production/applicationinsights.min.inc.js"
```
Comment (Deraen) [inline applicationinsights/boot-cljsjs-checksums.edn]: Hmm, this file should include both files. Run `boot package target` and check that `cljsjs` folder really has the both files, if not, sift move patterns might not be moving the files correctly.

File: applicationinsights/build.boot (hunk 2 above)
Comment (hkjels) [reply applicationinsights/build.boot]: Yeah. `0` is the minified version of the latest release. I'm not sure why it's like that

File: applicationinsights/boot-cljsjs-checksums.edn (hunk 3 above)
Comment (hkjels) [reply applicationinsights/boot-cljsjs-checksums.edn]: It works just fine on my box, but a colleague of mine is running into an issue on Windows with that sift-command he says

This is how it looks on my box:
```
.
├── common
│   └── applicationinsights.ext.js
├── development
│   ├── applicationinsights.inc.js
│   └── applicationinsights.inc.js.map
└── production
    ├── applicationinsights.min.inc.js
    └── applicationinsights.min.inc.js.map

3 directories, 5 files
```

I'll touch back when I know more about that error he's bumping into

Comment (hkjels) [reply applicationinsights/boot-cljsjs-checksums.edn]: Even without the `unzip` and the `sift move` tasks it doesn't run properly on windows. But he's not able to package and install other libraries in `cljsjs` either, so I'm thinking it's more of a problem with his box.
Will this be any different when it's pulled in as a dependency?

**Pull request merged**